CONTRIBUTING.md

```diff
@@ -234,3 +234,39 @@ run the following command and follow the prompts:
 ```
 yarn plop
 ```
+
+### Adding icons to Kaizen
+
+First check in with the team in the #prod_design_system channel - even if the icon already exists in the Figma UI Kit it's great to get a heads up so we can plan support.
+
+If you're given the all clear to add the icon follow these steps:
+
+- Create a new branch on the kaizen-design-system repo (ie: ${your-name}/add-pumpkin-icon) and move the icon SVG into `packages/component-library/icons` folder
+    - Make sure the file has a `.icon.svg` suffix
+- Create a PR (see the [contributing guide](https://cultureamp.design/components/overview)) and get it reviewed by a Design Systems Team member
```

Comment:
```suggestion
- Create a PR (see the [contributing guide](https://cultureamp.design/components/overview)) and get it reviewed by a Kaizen Team member
```

That's what we agreed on right? 😂

Reply:
Or "Kaizen Design System Team member" ;)

Also, that link to the `contributing guide` goes to the wrong place.